composer.json

```diff
@@ -33,8 +33,9 @@
         "zendframework/zend-stdlib": "^2.7.7 || ^3.0.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8 || ^5.0",
-        "squizlabs/php_codesniffer": "^2.3.1"
+        "phpunit/phpunit": "^4.8 || ^5.5",
+        "squizlabs/php_codesniffer": "^2.6.2",
+        "zendframework/zend-servicemanager": "^2.7.6 || ^3.1.1"
```

Comment:
Why is this added? I'm not seeing it used in any of the new code. If anything, I'd expect to see container-interop as a dev dependency...